service/matching/db_task_reader.go

```diff
@@ -48,6 +49,9 @@ type (
 		taskIterator(ctx context.Context, maxTaskID int64) collection.Iterator[*persistencespb.AllocatedTaskInfo]
 		ackTask(taskID int64)
 		moveAckedTaskID() int64
+		getBacklogCountHint() int64
+		getReadLevel() int64
+		getAckLevel() int64
```

Comment:
1. read / ack level is misleading, try to use read task ID / ack task ID
2. why do you need to expose this internal info to outside? if you need to calculate the backlog hint, then expose a function for backlog hint

Reply:
1. It is be consistent & compatible for taskqueuepb.TaskQueueStatus
2. taskqueuepb.TaskQueueStatus needs to be crafted in task queue manager.